managed/src/test/java/com/yugabyte/yw/commissioner/tasks/subtasks/KubernetesCommandExecutorTest.java

```diff
@@ -624,6 +624,64 @@ public void testHelmInstallForAnnotationsPrecendence() throws IOException {
     assertEquals(getExpectedOverrides(true), overrides);
   }
 
+  // https://github.com/yugabyte/yugabyte-db/issues/5930
```

Comment:
minor: don't need issue number here.
In the title of your commit, do include the `[#5930]` so it links to the right GH issue.

Reply:
Sure, I will include it in the body of the commit.